src/status_im/transport/message/v1/protocol.cljs

```diff
@@ -111,7 +111,7 @@
              :js-obj     (:js-obj cofx))]})
   (validate [this]
     (when (spec/valid? :message/message this)
-      (spec/conform :message/message this))))
+      this)))
```

Comment:
do not conform the message, it is already valid and conforming would introduce weird spec specific constructs.